core/camel-core/src/test/java/org/apache/camel/component/file/stress/FileAsyncStressReadLockNoneManualManualTest.java

```diff
@@ -17,15 +17,18 @@
 package org.apache.camel.component.file.stress;
 
 import java.util.Random;
+import java.util.concurrent.CountDownLatch;
 
-import org.apache.camel.Exchange;
-import org.apache.camel.Processor;
 import org.apache.camel.builder.RouteBuilder;
 import org.junit.jupiter.api.Disabled;
 
 @Disabled("Manual test")
 public class FileAsyncStressReadLockNoneManualManualTest extends FileAsyncStressManualTest {
```

Comment:
Same note about manual tests I wrote for `FileAsyncStressReadLockRenameManualManualTest`

Reply:
Thanks! 
I have reverted the changes.